Source/JavaScriptCore/jit/ExecutableAllocator.cpp

```diff
@@ -1364,7 +1365,7 @@ void dumpJITMemory(const void* dst, const void* src, size_t size)
                 return;
 
             needsToFlush = true;
-            flushQueue.get()->dispatchAfter(Seconds(Options::dumpJITMemoryFlushInterval()), [] {
+            flushQueueSingleton()->dispatchAfter(Seconds(Options::dumpJITMemoryFlushInterval()), [] {
```

Comment:
It doesn't seem to work since the Safer CPP bot is not telling you that you can un-skip this file.

Reply:
Oh! Actually it did. You just haven't updated the PR yet.

Reply:
I'm curious if it would work if we just renamed the variable to `flushQueueSingleton` instead of introducing a wrapper function (cc @rniwa).

Reply:
I thought about it too. But the original code isn't calling `dispatchAfter` on `flushQueue`, it's calling it on the Ref returned by get(). I imagine the `singleton` pattern to silence the warning only works if we call the function directly on the `singleton`. But yeah, perhaps Ryosuke can confirm.